test/NeoSharp.VM.Test/VMJsonTest.cs

```diff
@@ -0,0 +1,102 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace NeoSharp.VM.Test
+{
+    [TestClass]
+    public class VMJsonTest : VMJsonTestBase
+    {
+        [TestMethod]
+        public void TestJson()
+        {
+            var ut =
+@"
+{
+  ""Category"": ""ControlFlow"",
+  ""Name"": ""RET"",
+  ""Tests"":
+  [
+    {
+      ""Script"": ""Zg=="",
+      ""Actions"": 
+      [
+        {
+          ""Actions"": [""Execute""],
+          ""State"": 
+            {
+            ""State"": ""Halt"",
+            ""ConsumedGas"": 100000,
+            ""ResultStack"": [],
+            ""InvocationStack"": []
+            }
+        }
+      ],
+      ""Trigger"": ""Application""
+    }
+  ]
+}
+";
+
+            ExecuteTest(ut);
+
+            ut =
```

Comment:
Maybe, create json file and reference it?

Reply:
Yes, is just a sample, will create json files